### DISCUSSION TITLE
Why is the text in this PDF missing, on only one computer?

### DISCUSSION THREAD

**Discussion by CordeliaGS** [sumatrapdfreader/sumatrapdf#2977] [Q&A]
I have an odd problem. I opened a new RPG adventure PDF in SumatraPDF on my home computer, and happily read a few pages. later, at work, i wanted to keep reading; so I opened the _same_ file (downloaded from the same source, and then i sent the file from one computer to the other to be really sure) ~in the _same_ version of SPDF (3.4.6)~ (this was incorrect, the desktop was out of date.). On my laptop, all the text is missing; it's not visible, and also isn't selectable, as if that layer (or whatever, idk how it's structured under the hood) is just gone.

How the page appears on my laptop:
![eiRmfrt2cF](https://user-images.githubusercontent.com/22449828/191569092-b0543489-0938-4c2a-b6d0-d3c66ff5a88d.png)

How it should look (taken in Firefox, because I'm on the laptop rn, but it displays perfectly like this in SPDF on my home computer):

![SvzSEO8HcG](https://user-images.githubusercontent.com/22449828/191569217-6ad31767-28f5-458b-bd16-ed198dc51fd4.png)

Anyone know offhand what this might be? I can't upload the PDF as it's a copywritten book, but if there's no quick easy answer I'll actually open an issue report and email it in.

**Comment by GitHubRulesOK** [comment]
@DeclanGS 
Sorry without a sample this was unanswered However point was raised by another with a sample 
so your answer is here https://github.com/sumatrapdfreader/sumatrapdf/issues/2989#top
